frontend/src/components/House/index.js

```diff
@@ -0,0 +1,4 @@
+import House from './House';
+import HouseList from './HouseList';
+
+export {House, HouseList};
```

Comment:
what is happening here, do we need this?

Reply:
I think this is fine and generally a good practice, `House` is a feature and you are packing a feature into its own module.